web/scripts/generateZonesConfig.ts

```diff
@@ -37,6 +37,7 @@ const getConfig = (): CombinedZonesConfig => {
     'estimation_method',
     'parsers',
     'subZoneNames',
+    'aggregates_displayed',
```

Comment:
So all that fancy checking we had to see if an aggregate was disabled or not was kind of pointless without this. 🫠